c-example/README.md

```diff
@@ -0,0 +1,63 @@
+# C example
+
+This is the README file for spglib examples for C.
+
+## Examples of using spglib in a cmake project
+
+See the local [`CMakeLists.txt`](CMakeLists.txt) file for how to use
+spglib in a cmake project, using either native cmake options or pkg-config.
+
+## Examples of linking to spglib manually
+
+To manually compile `example.c`:
+
+```console
+gcc example.c -I[include directory] -L[library directory] -lsymspg -lm
+```
+
+where the `[include directory]` and `[library directory]` can be obtained
+via `pkg-config --cflags --libs spglib`.
+
+## Concept
```

Comment:
Can we move this to top level readme? I will suggest some re-wording such that someone clueless like me can understand.

Reply:
I have no clear view on the structure of README.md. Could it be possible for you to manage it after merging this PR?

Reply:
I mean at the `/README.md` instead of `c-example/README.md`

Reply:
I see.

Reply:
This was treated at 5800eb0,

Reply:
Yes, I like this one. I'll maybe simplify it later and move some back to the C api